### ISSUE TITLE
How do I enable Traefik routing for marathon containers

### ISSUE THREAD

**Issue by andreimc** [CiscoCloud/mantl#1543]
In traefik UI I don't see anything even tho I have the jupyter container running.

Should I not see that ? or is consul template required for that on traefik ?

I have an install using marathon:

```
---
# CHECK SECURITY - when customizing you should leave this in. If you
# take it out and forget to specify security.yml, security could be turned off
# on components in your cluster!
- include: "{{ playbook_dir }}/playbooks/check-requirements.yml"

# BASICS - we need every node in the cluster to have common software running to
# increase stability and enable service discovery. You can look at the
# documentation for each of these components in their README file in the
# `roles/` directory, or by checking the online documentation at
# docs.mantl.io.
- hosts: all
  vars:
    # consul_acl_datacenter should be set to the datacenter you want to control
    # Consul ACLs. If you only have one datacenter, set it to that or remove
    # this variable.
    # consul_acl_datacenter: your_primary_datacenter

    consul_servers_group: role=control
  roles:
    - common
    - lvm
    - docker
    - logrotate
    - consul-template
    - nginx
    - consul
    - etcd
    - dnsmasq

# ROLES - after provisioning the software that's common to all hosts, we do
# specialized hosts. This configuration has two major groups: control nodes and
# worker nodes. We provision the worker nodes first so that we don't create any
# race conditions. This could happen in the Mesos components - if there are no
# worker nodes when trying to schedule control software, the deployment process
# would hang.
#
# The worker role itself has a minimal configuration, as it's designed mainly to
# run software that the Mesos leader shedules. It also forwards traffic to
# globally known ports configured through Marathon.
- hosts: role=worker
  # role=worker hosts are a subset of "all". Since we already gathered facts on
  # all servers, we can skip it here to speed up the deployment.
  gather_facts: no
  vars:
    mesos_mode: follower
    zookeeper_server_group: role=control
  roles:
    - mesos

# the control nodes are necessarily more complex than the worker nodes, and have
# ZooKeeper, Mesos, and Marathon leaders. In addition, they control Vault to
# manage secrets in the cluster. These servers do not run applications by
# themselves, they only schedule work. That said, there should be at least 3 of
# them (and always an odd number) so that ZooKeeper can get and keep a quorum.
- hosts: role=control
  gather_facts: yes
  vars:
    consul_servers_group: role=control
    mesos_leaders_group: role=control
    mesos_mode: leader
    zookeeper_server_group: role=control
  roles:
    - vault
    - zookeeper
    - mesos
    - marathon
    - mantlui

# The edge role exists solely for routing traffic into the cluster. Firewall
# settings should be such that web traffic (ports 80 and 443) is exposed to the
# world.
- hosts: role=edge
  gather_facts: yes
  vars:
    # this is the domain that traefik will match on to do host-based HTTP
    # routing. Set it to a domain you control and add a star domain to route
    # traffic. (EG *.marathon.localhost)
    #
    # For those migrating from haproxy, this variable serves the same purpose
    # and format as `haproxy_domain`.
    traefik_marathon_domain: marathon.localhost
  roles:
    - traefik
```

- Git commit hash or branch: 1.1.0
- Cloud Environment: AWS

**Comment by ygoodmn** [comment]
Would suggest you post your DCOS Json for "jupyter container" as we put in labels in the POST to Marathon and we see this in Traefik:
Example:
"labels": {
  "traefik.frontend.rule": "PathPrefix",
  "traefik.frontend.value": "/pathtouseforfrontend/"
        },
Or as seen in UI of Marathon Configuration with multiple paths
Labels
traefik.frontend.rule=PathPrefix
traefik.frontend.value=/{reg:(path1|path/path2|path/path/path3)}

**Comment by ryane** [comment]
The marathon labels that can be used to configure traefik can be found here: https://docs.traefik.io/toml/#marathon-backend

If your marathon app is configured correctly for traefik and is still not available, you might check the traefik logs on the edge node(s) to see if there are any errors.

**Comment by andreimc** [comment]
@ygoodmn i was meaning this one https://github.com/CiscoCloud/mantl/blob/master/examples/jupyter/jupyter.json

what I would like is the route to be: jupyter.example.com where example.com is the domain I am using.

Not exactly sure what to use but will give it a go :) I think the examples should be a little bit more comprehensive but if I figure it out I will do a PR :)


@ryane cool I will give it a go 👍

**Comment by andreimc** [comment]
My problem was because the ws fixed by this PR #1547
